src/mlpack/bindings/cli/add_to_po.hpp

```diff
@@ -2,110 +2,117 @@
  * @file add_to_po.hpp
  * @author Ryan Curtin
  *
- * Utility functions to add options to boost::program_options based on their
+ * Utility functions to add options to CLI11 based on their
  * type.
```

Comment:
```suggestion
 * Utility functions to add options to CLI11 based on their type.
```

This can be reflowed. :)